components/Editor.js

```diff
@@ -425,6 +449,14 @@ class Editor extends React.Component {
             </Overlay>
           )}
         </Dropzone>
+        <Box marginTop="1rem" display="flex" justifyContent="center">
+          <Toolbar>
+            <AspectRatioButtons
```

Comment:
I think this toolbar needs to be in the settings, under "Settings (Gear Icon) > Window"